rdt-app/src/main/java/io/ona/rdt_app/contract/CustomRDTCaptureContract.java

```diff
@@ -0,0 +1,15 @@
+package io.ona.rdt_app.contract;
+
+/**
+ * Created by Vincent Karuri on 14/08/2019
+ */
+public interface CustomRDTCaptureContract {
```

Comment:
This interface does not define any methods

Reply:
Yes, none are needed for now.

It's just following mvp architecture.